src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const NORMALIZED_ROBOT_OBJECT = {};
+  const CONTROL_IF_KEY_DUBBED = [];
+
+  for (const key in robot) {
+    const RIGHT_WAY_ROBOT_KEY = robot[key];
+    const RIGHT_WAY_ROBOT_VALUE = key;
+
+    NORMALIZED_ROBOT_OBJECT[RIGHT_WAY_ROBOT_KEY] = RIGHT_WAY_ROBOT_VALUE;
+
+    if (!CONTROL_IF_KEY_DUBBED.includes(RIGHT_WAY_ROBOT_KEY)) {
+      CONTROL_IF_KEY_DUBBED.push(RIGHT_WAY_ROBOT_KEY);
+    } else {
+      return null;
+    }
+  }
+
+  return NORMALIZED_ROBOT_OBJECT;
```

Comment:
You can rewrite like this, consider it will be better and easier
```suggestion
  const inversedRobot = {};

  for (const key in robot) {
    const inversedKey = robot[key];

    if (inversedRobot.hasOwnProperty(inversedKey)) {
      return null;
    }
    inversedRobot[inversedKey] = key;
  }

  return inversedRobot;
```